index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FCC: Tribute Page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1>
+      <p>Человек, который изменил мир.</p>
+      <div id="img-div">
+        <img id="image" src="stive.jpg" alt="Steve Jobs" />
+        <figure id="img-caption">
+          Я не доверяю компьютеру, который не могу поднять (Стив Джобс).
+        </figure>
+      </div>
+      <section id="tribute-info">
+        <h3 id="heading">Краткая биография Стива Джобса:</h3>

```

Comment:
Where is h2 in you code? Why do you use h3 here, right after h1?

Reply:
Corrected!

Reply:
My idea was in adding class to the date. But your solution could also take place.